apps/website/.vuepress/theme/styles/overrides.scss

```diff
@@ -289,6 +289,10 @@ body .no-clr-ui p {
   //margin-bottom: 0;
 }
 
+[cds-text~='body'] li {
+  color: var(--clr-p1-color);
```

Comment:
is there a `--cds-token` we can use here instead?

Reply:
I wanted to map this to the paragraphs around it, so this seems like the best one?

Reply:
The problem here is that the `--clr-p?-color` custom properties are from Clarity UI. I think we want to limit our dependency there if possible. If you want to assign the color with an exact match in the Core design tokens, the color you want is `--cds-global-color-gray-900`. I did the digging to find what that p1's color value was and `gray-900` is a 100% match.